kolibri/deployment/management/commands/startnginx.py

```diff
@@ -0,0 +1,154 @@
+import logging as logger
+import os
+import signal
+import tempfile
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+
+try:
+    import subprocess
+except ImportError:
+    subprocess = None
+
+# from kolibri.utils.conf import OPTIONS
+
+logging = logger.getLogger(__name__)
+
+NGINX_CONF_PATH = "/etc/nginx/sites-enabled/007-kolibri"
+
+context = {
+    "kolibri_home": settings.KOLIBRI_HOME,
+    # "http_port": OPTIONS["Deployment"]["HTTP_PORT"],
+    "http_port": os.environ.get("KOLIBRI_LISTEN_PORT", 8080),
+    "static_root": settings.STATIC_ROOT,
+    "content_database_dir": settings.CONTENT_DATABASE_DIR,
+    "content_storage_dir": settings.CONTENT_STORAGE_DIR,
+}
+
+nginx_config = """
```

Comment:
Contents of the UWSGI config also live and be written to a temporary named file? Or be set as command line args?